gl/gloffscreen_egl.c

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
Does this EGL code actually work?

If you haven't tested it, feel free to get rid of it.